### PR TITLE
feat: Add AMPE price to the Dashboard

### LINKED ISSUE #258: Add AMPE price to the dashboard
### Context
AMPE is now live on Zenlink and has price, It would be good if we start showing the AMPE price on the Portal dashboard. 

## Requirement
* Show AMPE price on Portal dashboard
* How to get AMPE price?
    * Query the [squid](https://squid.subsquid.io/amplitude-squid/graphql) 
    * Here is the sample query

```
query MyQuery {
  bundleById(id: "1") {
    ethPrice
  }
}
```

Also mentioned my @ebma in the below [comment](https://github.com/pendulum-chain/portal/issues/258#issuecomment-1739696968)

### PR DESCRIPTION
**Pull request by gonzamontiel**
Closes #258

### CODE REVIEW

File: src/hooks/usePriceFetcher.ts [hunk 1]
```diff
@@ -66,8 +68,8 @@ const assets: PriceFetcherAsset[] = [
   {
     assetName: 'AMPE',
     blockchain: 'Amplitude',
-    assetId: undefined,
-    provider: 'dia',
+    assetId: 'eth',
```
Comment (ebma) [inline src/hooks/usePriceFetcher.ts:71]: No need to call it `eth`, we can also call it 'ampe' here I guess.

**Review by ebma — APPROVED** [review]
LGTM

File: src/hooks/usePriceFetcher.ts (hunk 1 above)
Comment (gonzamontiel) [reply src/hooks/usePriceFetcher.ts:71]:
```suggestion
    assetId: 'eth',
    assetId: 'ampe',
```

**Pull request merged**